nexus/catalog/migrations/V30__more_dynconf_settings.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO dynamic_settings (config_name,config_default_value,config_value_type,config_description,config_apply_mode)
```

Comment:
these migration files are why I'd rather have envs & defaults in code, not db. In particular this migration script fails if we add an env & insert it into table causing conflict